App.js

```diff
@@ -1,26 +1,22 @@
 import React from 'react'
+import data from './data.json'
```

Comment:
Let's remember that `data` is local dependency, therefore it should be placed below outer dependencies